x-pack/plugins/reporting/server/export_types/csv/server/generate_csv/field_format_map.ts

```diff
@@ -37,12 +30,13 @@ export function fieldFormatMapFactory(
     const fieldFormatMap = JSON.parse(indexPatternSavedObject.attributes.fieldFormatMap);
     Object.keys(fieldFormatMap).forEach((fieldName) => {
       const formatConfig: FieldFormatConfig = fieldFormatMap[fieldName];
+      const formatParams = {
+        ...formatConfig.params,
+        timezone,
+      };
 
       if (!_.isEmpty(formatConfig)) {
-        formatsMap.set(
-          fieldName,
-          fieldFormatsRegistry.getInstance(formatConfig.id, formatConfig.params)
-        );
+        formatsMap.set(fieldName, fieldFormatsRegistry.getInstance(formatConfig.id, formatParams));
```

Comment:
NOTE: This is not yet quite working as intended. It's not clear if this actually uses the given timezone to format any of the date strings.  This is the server context and things on the appArch side have changed since I last contributed this PR: https://github.com/elastic/kibana/pull/30264/

cc @mattkime